app/views/repairs/index.html.erb

```diff
@@ -0,0 +1,11 @@
+| <%= link_to "New Repair", new_repair_path(@repair) %> |
+<% @repair.each do |repair| %>
+    <div>
+    <%= repair.repair_type %>
+    </div>
+    <div>
+    <%= repair.repair_notes %>
+    </div>
+     <%= link_to "Edit", edit_repair_path(repair) %>
+     <%= link_to "Delete",repair_path(repair), method: :delete%>
```

Comment:
To follow best style practices, you need to add a space before the closing `%>` and after the first comma.